### ISSUE TITLE
Sync: Invalid Gist ID

### ISSUE THREAD

**Issue by M4r28** [shanalikhan/code-settings-sync#1070]
I've set up a gist and uploaded settings successfully. Although then faced errors when trying to follow the next steps and downloading it on the new machine.  
I then deleted the gist and tried to start over after uninstalling the extension and restarting the machine and vscode. Now when trying to upload settings I keep getting the 'Sync: Invalid Gist ID' error. 
Any idea how to solve it?

**Comment by shanalikhan** [comment]
Try running `Sync : Reset Extension Settings` and restart code to setup Settings Sync.
If you still face issue, post your developer console log

**Comment by solirian** [comment]
"I've set up a gist and uploaded settings successfully. Although then faced errors when trying to follow the next steps and downloading it on the new machine.
I then deleted the gist and tried to start over after uninstalling the extension and restarting the machine and vscode. Now when trying to upload settings I keep getting the 'Sync: Invalid Gist ID' error.
Any idea how to solve it?" 

Same Problems the last to days 

but i know what i most du now ;>
